corpus.conllulex

```diff
@@ -6438,7 +6438,7 @@
 # text = The > vs > = is definitely something you can just write and forget about , and when you re testing if you re testing on an account that has like 200 wins , you wo n't see it .
 1	The	the	DET	DT	_	3	det	_	_	_	_	_	_	_	_	_	_	_
 2	>	>	PUNCT	-RRB-	_	3	punct	_	_	_	_	_	_	_	_	_	_	_
-3	vs	versus	NOUN	NN	_	8	nsubj	_	_	_	_	_	`c	`c	_	_	_	_
+3	vs	versus	ADP	IN	_	8	nsubj	_	_	_	_	_	`c	`c	_	_	_	_
```

Comment:
I think this one is better as CCONJ/CC (and that's consistent with what the supersense annotators thought). [discussion of *versus* grammaticalized as a conjunction](https://www.researchgate.net/profile/Brett-Reynolds-2/publication/260122411_New_members_of_%27closed_classes%27_in_English/links/0f31752fa227778d53000000/New-members-of-closed-classes-in-English.pdf)